modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordProjectService.kt

```diff
@@ -17,7 +17,7 @@ class MirrordProjectService(val project: Project) : Disposable {
 
     val versionCheck: MirrordVersionCheck = MirrordVersionCheck(this)
 
-    val mirrordApi: MirrordApi = MirrordApi(this)
+    val mirrordApi: MirrordApi = MirrordApi(this, null)
```

Comment:
```suggestion
    fun mirrordApi(environment: Map<String, String>): MirrordApi {
        return MirrordApi(this, environment)
    }
```

I think you still use this field to run the CLI, so you don't actually use the complete env passed to `MirrordExecManager.start`. See line 124 of the `MirrordExecManager`